stroom-core-client/src/main/java/stroom/data/client/presenter/MetaPresenter.java

```diff
@@ -128,6 +129,9 @@ public MetaPresenter(final EventBus eventBus,
             streamRelationListRestore.setTitle("Restore");
         }
 
+        // Selection information
+        streamListInfo = metaListPresenter.add(SvgPresets.INFO);
```

Comment:
I would set the button title to something like `Selection Summary` to distinguish it from the Info sub-tab of the bottom pane.